docs/integration/java/Pulsar-manager.md

```diff
@@ -20,21 +20,21 @@ After a successful deployment, you need to ensure:
 - Open your favorite browser and visit **http://localhost:7001**, you will see the login page of Casdoor.
 - Input `admin` and `123` to test login functionality is working fine.
 
-Then you can quickly implement a casdoor based login page in your own app with the following steps.
+Then you can quickly implement a casdoor-based login page in your own app with the following steps.
 
 ## Step2. Configure Casdoor 
-Configure casdoor can refer to [casdoor](https://door.casdoor.com/login)(Configure casdoor's browser better not use one browser with your develop browser).
+Configure casdoor can refer to [casdoor](https://door.casdoor.com/login)(Configure casdoor's browser better not use one browser as your develop browser).
```

Comment:
@jakiuncle plz review this.
do you mean:
> It is better not to use the same browser to configure and develop Casdoor

Reply:
yes

Reply:
In fact, I didn't understand what the author was trying to say.